domain/src/commonMain/kotlin/com/islamversity/domain/model/JuzRepoModel.kt

```diff
@@ -0,0 +1,8 @@
+package com.islamversity.domain.model
+
+data class JuzRepoModel(
+    val ayaId: String,
+    val surahId: String,
+    val juz: Long,
+    val hizb: Boolean
```

Comment:
what does this boolean mean?
hizb is also a numerb

Reply:
These models & mappers after DB are just temporary! I'm not sure, what data I'll need?

Reply:
NP, please just remove the Hizb
We don't need it